Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -84,6 +90,20 @@ static inline D2D_RECT_F __CGRectToD2D_F(CGRect rect) {
 
     // Alpha Blending
     CGFloat alpha = 1.0f;
+
+    inline void ComputeStrokeStyle(ID2D1Factory* factory) {
+        if (strokeStyle) {
+            return;
+        }
+
+        std::vector<CGFloat> adjustedDashes(dashes.size());
+        std::transform(dashes.cbegin(), dashes.cend(), adjustedDashes.begin(), [this](const float& f) -> CGFloat { return f / lineWidth; });
```

Comment:
actually, there is; but i have the direction backwards.

CGFloat flips between float and double in certain reference platform configurations, but D2D prefers float. Since we're passing a raw buffer, we must type-match.

I should be storing CGFloats internally and converting to float here.

---
In reply to: [84096583](https://github.com/Microsoft/WinObjC/pull/1181#discussion_r84096583) [](ancestors = 84096583)